wear/wear.iml

```diff
@@ -30,6 +31,7 @@
     <output-test url="file://$MODULE_DIR$/build/intermediates/classes/androidTest/xdrip/debug" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
+<<<<<<< HEAD:wear/wear.iml
```

Comment:
OK, I have hit a snag.  wear/wear.iml needs for some reason to be here.  If
I git rm --cached, it deletes it from the branch, which I believe will
delete it from master if this is merged.  Is that the case?
I really don't want to do that.  I will check it out from master again
prior to committing and pushing PebbleTrend, to remove the merge
conflicts.  Once it is in, I will make sure I never add an iml file to my
commits.
Cheers

On Thu, Jun 16, 2016 at 7:18 AM, John Stevens <jstevensog@gmail.com> wrote:

> OK, guys.  Thanks so much for your feedback.
> I have checked out wear/wear.iml, app/build.gradle, and
> app/src/main/res/xml/pref_notifications.xml from master to remove those
> conflicts.  I have modified utils/Preferences.java to remove the null
> pointer exceptions I got a couple of days ago.
>
> I have left the AlarmClock check in Notifications.java for now.  It hasn't
> solved the down time issue at all, but I thought it prudent to leave it
> there until we have a better idea.
> Will do a build and test run and update the PR.
> Thanks again and Cheers
>
> On Thu, Jun 16, 2016 at 2:13 AM, tzachi-dar <notifications@github.com>
> wrote:
>
>> In wear/wear.iml
>> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/256#discussion_r67193882>
>> :
>>
>> > @@ -30,6 +31,7 @@
>> >      <output-test url="file://$MODULE_DIR$/build/intermediates/classes/androidTest/xdrip/debug" />
>> >      <exclude-output />
>> >      <content url="file://$MODULE_DIR$">
>> > +<<<<<<< HEAD:wear/wear.iml
>>
>> There are merge conflicts here, but I don't think you need to submit this
>> file in any case.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/256/files/a063e56a27b0e35ee4f36ddb0ed7af3f50a9d3a7#r67193882>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AIQs8zWRScxJE-hQexij5j0-QXkksUSdks5qMCSngaJpZM4HHlcy>
>> .
>>
>
>
>
> --
> John Stevens
> "You are how you live, not what you have."
>



-- 
John Stevens
"You are how you live, not what you have."